bedrock/firefox/templates/firefox/geolocation.html

```diff
@@ -201,6 +201,20 @@ <h3 data-accordion-role="tab">{{ _('How do I turn off Location-Aware Browsing pe
   </div>
 </section>
 
+<section>
+  {% if l10n_has_tag('geolocation_faq_update') %}
```

Comment:
Should i indent the <section> block after I wrapped the I10n tag outside the <section> element?